repokitteh.star

```diff
@@ -1,3 +1,3 @@
-use("github.com/softkitteh/repokitteh-modules/assign.star")
-use("github.com/softkitteh/repokitteh-modules/review.star")
-use("github.com/softkitteh/repokitteh-modules/wait.star")
+use("github.com/repokitteh/modules/assign.star")
```

Comment:
@itayd one thought here for a future PR. I think when we import modules into our repo we should do it by SHA so that we can clearly track what code we are using? Thoughts on doing that in a follow up? cc @htuch

Reply:
@mattklein123 sure thing. for now it's easier to refactor things behind the scenes without causing interruption, but i'll pin the shas in next pr.